frontend/corrections.py

```diff
@@ -20,10 +19,10 @@
 }
 
 corrections_blueprint = Blueprint('corrections', __name__,
-                                  template_folder='templates')
+                                  template_folder='templates', url_prefix='/corrections')
 
 
-@corrections_blueprint.route("/corrections/<string:queue_name>", methods=["GET"])
+@corrections_blueprint.route("/<string:queue_name>", methods=["GET"])

```

Comment:
please separate formatting/refactoring changes in a separate pull request than the actual functional change.

Reply:
Gotcha!

Reply:
Reverted back